app/packages/looker/src/elements/common/tags.ts

```diff
@@ -40,6 +40,7 @@ import { lookerTags } from "./tags.module.css";
 interface TagData {
   color: string;
   title: string;
+  path: string;
```

Comment:
Nice~ 
I saw that you were using the tags to validate tooltip on hover. 
if we want to verify the tags/detections are correct on looker and on flash light, in that case, would screenshots be better? Or this is still the preferred pattern?

Reply:
Rule of thumb is to reserve screenshots only for elements that are not in the DOM.